relayer/cmd/getblock.go

```diff
@@ -0,0 +1,198 @@
+// Copyright 2020 Snowfork
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package cmd
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/ethclient"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+
+	gethTypes "github.com/ethereum/go-ethereum/core/types"
+	"github.com/snowfork/polkadot-ethereum/relayer/chain/ethereum"
+	"github.com/snowfork/polkadot-ethereum/relayer/core"
+)
+
+type Format string
+
+const (
+	RustFmt Format = "rust"
+	JSONFmt Format = "json"
+)
+
+func getBlockCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "getblock",
+		Short:   "Retrieve the latest finalized block",
```

Comment:
Would it be useful to extend this command for Polkadot blocks as well @vgeddes?

Reply:
yeah definitely, it would be useful :+1:

Though currently our ethereum contracts aren't storing the latest finalized block, so we can query for that info yet